File: tournaments/isIncreasingDigitsSequence/isIncreasingDigitsSequence.js
function isIncreasingDigitsSequence(n) {
    let digit = n%10;
    n = ~~(n/10);

    while (0 < n) {
        if (digit <= n%10) return false;
        digit = n%10;
        n = ~~(n/10);
    }

    return true;
}

File: tournaments/isIncreasingDigitsSequence/isIncreasingDigitsSequence.cpp
bool isIncreasingDigitsSequence(int n) {
  int last = 10;
  while (n != 0) {
    if (n % 10 >= last) {
      return false;
    }
    last = n % 10;
    n /= 10;
  }
  return true;
}
